pixi.toml

```diff
@@ -115,6 +122,7 @@ docs = { features = [
 lint = { features = [
   "lint",
 ], no-default-feature = true, solve-group = "default" }
+pypi-gen = { features = ["pypi-gen"] }
```

Comment:
Would it make sense to add solve_group "default" here?

Reply:
What would be the reason to do that? Its just a script to generate some files so it doesnt need to share any of the dependencies with other environments.

Reply:
Fair enough.